d2l-button-shared-styles.html

```diff
@@ -1,6 +1,12 @@
 <link rel="import" href="../polymer/polymer.html">
 <style is="custom-style">
 	html {
+		--d2l-button-clear-focus: {
+			box-shadow: 0 0 0 4px rgba(0, 0, 0, 0);
+		};
```

Comment:
What happens without the `--d2l-button-clear-focus`? I'm just curious why it's necessary.